bin.src/validateCfht.py

```diff
@@ -25,15 +25,12 @@
 import os.path
 import sys
 
-from lsst.validate.drp import checkAstrometry
+from lsst.validate.drp import checkAstrometryPhotometry
 
 
 def defaultData(repo):
     # List of visits to be considered
-    visits = [850587]
-
-    # Reference visit (the other viisits will be compared to this one
-    ref = 849375
+    visits = [849375, 850587]
```

Comment:
This list of visits & CCDs seems to simply be a repetition of `examples/runCfht.list`. It would be better to define it once.